Documentation/markdown/SoftwareArchitecture.md

```diff
@@ -15,63 +16,119 @@
+This point cloud module has references to Qt5, Liblas, Eigen, and CloudCompare ([more details](Licenses/readme.md)).
+
+By default, the _Point Cloud Processing_ module is deactivated. 
+In order to make use of it, OIP needs to be built with the corresponding `CMake` option selected 
+(for further details, see [CMake Options Documentation](Documentation/markdown/CMakeOptions.md)). 
+A significant advantage is that the _Core_ and the _Point Cloud Processing_ module can be run simultaneously 
+so that, e.g., one can open an IFC file and a point cloud 
+at the same time to compare them to each other directly.
 
 ### <a name="UI"></a> User Interface
 
-The underlying part of the user interface is the Rendering module. This contains a self-written rendering engine called BlueFramework. Depending on the objects that are to be displayed, different types of shaders and effects are implemented. The Graphical User Interface is built on top of this, which has a 3D viewport to display geometric objects and several menus and buttons that provide functions to interact with them. The GUI bases on the Qt library, which has an extensive catalog of ready-to-use user interface elements. Other third-party libraries that the GUI uses are:
-* Zlib (algorithms for compression and decompression of data)
-* Proj (functionalities to calculate conversions between geodetic reference systems)
+The underlying part of the user interface is the _Rendering module_. 
+This links to a self-written rendering engine called _BlueFramework_. 
+Depending on the objects that are to be displayed, different types of shaders and effects are implemented. 
+The _Graphical User Interface_ is built on top of this, 
+which has a 3D viewport to display geometric objects and several menus and buttons that provide functions to interact with them. 
+The GUI bases on the `Qt` library, which has an extensive catalog of ready-to-use user interface elements. 
+Other third-party libraries that the GUI uses are Zlib and Proj (for more details see [here](Licenses/readme.md))
 
-On the other hand, there is the Unit Testing part, which bases on Google Test. Since many unit tests that are implemented in the OIP use visual comparison, the unit test base includes a geometry model renderer that allows rendering without the use of the Graphical User Interface. On top of that, there are various individual unit tests for IFC, OFF, and point clouds (and also some other more general unit tests?). These provide automated checking to guarantee that implemented functionalities are all still in place and working as expected. For the case of the IFC format, the [List of supported IFC Representations](./SupportedIFCrepresentations.md) provides information about which representations currently have at least one unit test in place.
+### <a name="UT"></a> Unit Testing
+
+On the other hand, there is the _Unit Testing_ part, which bases on `Google Test`. 
```

Comment:
Careful - if one uses *on the other hand*  there **must** be *on the one hand* somewhere as well.

Reply:
Resolved: 7aaecb6